compose/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Wed Oct 04 10:29:43 EDT 2023
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
Oooh, we are trying with gradle 8.0 now? 🤩

Reply:
8.0 seems to work fine for me. Downgrading it doesn't work.